modules/virt-enabling-preallocation-for-dv.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virtual_disks/virt-using-preallocation-for-datavolumes.adoc
+
+[id="virt-enabling-preallocation-for-dv_{context}"]
+= Enabling preallocation for a data volume
+
+If preallocation is not globally enabled for the cluster, you can enable it for specific data volumes by including the `spec.preallocation` field in the data volume manifest. This can be done in either the web console or when using the OpenShift client (oc).
```

Comment:
I'd change the second sentence to the active voice: "You can enable it in either the web console or by using the OpenShift client (`oc`)."

I think oc should be in backticks regardless